teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor @specialization
+
```

Comment:
- [ ] Please remove this line `attr_accessor @specialization` because the specialization variable already is an initialization variable so the teacher class already has access to it.